src/main/features/response/form/models/howMuchPaid.ts

```diff
@@ -0,0 +1,73 @@
+import { Serializable } from 'models/serializable'
+import { IsDefined, IsPositive, MaxLength, Min } from 'class-validator'
+import { IsPastDate } from 'forms/validation/validators/datePastConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { IsValidYearFormat } from 'app/forms/validation/validators/isValidYearFormat'
+import { MomentFactory } from 'common/momentFactory'
+import { MomentFormatter } from 'app/utils/momentFormatter'
+import { Fractions } from 'forms/validation/validators/fractions'
+
+const currentDate = MomentFormatter.formatLongDate(MomentFactory.currentDate())
+
+export class ValidationErrors {
+  static readonly NOT_OWE_FULL_AMOUNT_REQUIRED: string = 'Explain why you don’t owe the full amount'
+  static readonly REASON_NOT_OWE_MONEY_TOO_LONG: string = 'Enter reason no longer than $constraint1 characters'
+  static readonly VALID_AMOUNT_REQUIRED: string = 'Enter a valid amount paid'
+  static readonly AMOUNT_REQUIRED: string = 'Enter an amount'
+  static readonly AMOUNT_NOT_VALID: string = 'Enter valid amount'
```

Comment:
A `AMOUNT_NOT_VALID` and `VALID_AMOUNT_REQUIRED` are pretty much the same, why we are having two very similar messages?